packages/docs/src/examples/Transaction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { parseEther } from '@ethersproject/units'
+import { Mainnet, DAppProvider, useSendTransaction, useEthers, Config } from '@usedapp/core'
+import { ChainId } from '../../../core/src/constants/chainId'
+
+const config: Config = {
+    readOnlyChainId: Mainnet.chainId,
+    readOnlyUrls: {
+      [Mainnet.chainId]: 'https://mainnet.infura.io/v3/57fc2c19095745e59ab96a4aa87dada8',
+    },
+}
+
+ReactDOM.render(
+    <DAppProvider config={config}>
+      <TokenBalance />
+    </DAppProvider>,
+    document.getElementById('root')
+)
+
+export function TokenBalance() {
+    const { chainId } = useEthers()
+    const { sendTransaction, state } = useSendTransaction()
+    
+    const status = state.status
+    const disabled = chainId === ChainId.Mainnet
```

Comment:
```suggestion
    // We prevent the example from running on Mainnet so that the users do not use real Ether without realizing.
    const disabled = chainId === ChainId.Mainnet
```

Always good to provide some explanation for non-obvious things.